nfldfs/constants.py

```diff
@@ -0,0 +1,3 @@
+CURRENT_SEASON = 2020
```

Comment:
#reminder: write a function to determine the current season

Reply:
Will move this to a new PR.